sdk/core/azure-core-amqp/src/models/amqp_properties.cpp

```diff
@@ -296,7 +296,16 @@ namespace Azure { namespace Core { namespace Amqp { namespace Models {
     std::string timeToString(std::chrono::system_clock::time_point t)
     {
       std::time_t time = std::chrono::system_clock::to_time_t(t);
+#ifdef _MSC_VER
```

Comment:
Can we use the save versions instead?

Reply:
I'm going to try that as an experiment - cppreference claims it's been a crt requirement from c++ 11 so it theoretically shoudl be there.